hw-latest/fpga/FPGA_Instructions.md

```diff
@@ -0,0 +1,91 @@
+_*SPDX-License-Identifier: Apache-2.0<BR>
+<BR>
+<BR>
+Licensed under the Apache License, Version 2.0 (the "License");<BR>
+you may not use this file except in compliance with the License.<BR>
+You may obtain a copy of the License at<BR>
+<BR>
+http://www.apache.org/licenses/LICENSE-2.0 <BR>
+<BR>
+Unless required by applicable law or agreed to in writing, software<BR>
+distributed under the License is distributed on an "AS IS" BASIS,<BR>
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.<BR>
+See the License for the specific language governing permissions and<BR>
+limitations under the License.*_<BR>
+
+# **Caliptra FPGA Guide** #
+
+### Requirements: ###
+ - Vivado
+   - `Version v2022.2`
+ - FPGA
+   - `ZCU104 Development Board`
+   - https://www.xilinx.com/products/boards-and-kits/zcu104.html
+ - [Not fully enabled] JTAG debugger
+   - `Olimex ARM-USB-TINY-H`
+
+### ZCU104 Processing System One Time Setup: ###
+1. Install ZCU104 SD card image
+   - https://ubuntu.com/download/amd-xilinx
+1. Configure SW6 to boot from SD1.
+   - Mode SW6[4:1]: OFF, OFF, OFF, ON
+1. Install rustup using Unix directions: https://rustup.rs/#
+1. apt install libssl-dev
+
+#### Serial port configuration ####
+Serial port settings for connection over USB.
+ - Speed: 115200
+ - Data bits: 8
+ - Stop bits: 1
+ - Parity: None
+ - Flow control: None
+
+### FPGA Build Steps: ###
+1. Launch Vivado with tcl script defining FPGA build
+    - `vivado -mode batch -source fpga_configuration.tcl`
+1. Run Synthesis
+    - Use GUI or: `launch_runs synth_1`
```

Comment:
I think we need to make it clearer how to script this, and run it with parallelization

```
launch_runs synth_1 -jobs 10
wait_on_runs synth_1
launch_runs impl_1 -jobs 10
wait_on_runs impl_1
```